biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.tsx

```diff
@@ -36,6 +45,10 @@ export const InputPage = () => {
       endDate,
       smallBatteryCells: parseInt(smallBatteryCells as string),
       largeBatteryCells: parseInt(largeBatteryCells as string),
+      numBuildings: parseInt(numBuildings as string),
```

Comment:
why is the input above labeled with type 'number' but here its cast as a string then parsed as an int?

Reply:
Thanks didn't catch that.